tests/test_vms_exist.py

```diff
@@ -41,6 +41,7 @@ def test_sd_whonix_config(self):
         self.assertTrue(vm.template == "sd-whonix-template")
         self.assertTrue(vm.provides_network)
         self.assertFalse(vm.template_for_dispvms)
+        self.assertTrue('sd-workstation' in vm.tags)
```

Comment:
:ok_hand: Great addition; keeping these tests will help in confirming that the `cmd.run` -> `qvm.tags` conversion suggested above works well.